app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -142,9 +145,14 @@ function DocumentCapture({ isAsyncForm = false, onStepChange }: DocumentCaptureP
         },
       ].filter(Boolean) as FormStep[]);
 
+  const stepIndicatorPath =
+    stepName && ['location', 'prepare', 'switch_back'].includes(stepName)
```

Comment:
it would be cool if we had more static steps so we could do like `LocationStep.name`, `PrepareStep.name`, `SwitchBackStep.name` etc but looking at the step definitions, that is definitely a task for another time

Reply:
> it would be cool if we had more static steps so we could do like `LocationStep.name`, `PrepareStep.name`, `SwitchBackStep.name` etc but looking at the step definitions, that is definitely a task for another time

Hm, yeah, I can see how that would be nice to have. Technically the step definitions are already an object from which we could reference `step.name`, and in the FSMv2 flow we do [treat those objects as first-class exports](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/verify-flow/steps/personal-key/index.ts), but we haven't really done that here. It could also be nice to collapse it so that the properties are assigned to the React component, but that may add some confusion, and historically I've had a hard time with TypeScript tolerating custom properties on functions.